### PR TITLE
Call out the supported Amazon EMR release versions for DJM

### PR DESCRIPTION
**Pull request by yiliangzhou**
### What does this PR do? What is the motivation?

It's not clear to customer what Amazon EMR release version the Data Jobs Monitoring support at the moment, this is causing confusion and unsuccessful setup for customers. This PR is to call out the supported versions: Amazon EMR 6.6.0 and above.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/data_jobs/emr.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ further_reading:
 
 ## Setup
 
-Follow these steps to enable Data Jobs Monitoring for Amazon EMR.
+Follow these steps to enable Data Jobs Monitoring for Amazon EMR. [Amazon EMR Release 6.6.0][10] and above are supported.
```
Comment (hestonhoffman) [inline content/en/data_jobs/emr.md:14]:
```suggestion
Follow these steps to enable Data Jobs Monitoring for Amazon EMR.
```

File: content/en/data_jobs/emr.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ further_reading:
 
 ## Setup
```
Comment (hestonhoffman) [inline content/en/data_jobs/emr.md:12]:
```suggestion
## Requirements

[Amazon EMR Release 6.6.0][10] or later is required.

## Setup
```

**Review by hestonhoffman — CHANGES_REQUESTED** [review]
I made some suggestions to make this more consistent with the way we usually document requirements.

**Review by warrierr — APPROVED** [review]
LGTM

**Review by hestonhoffman — APPROVED** [review]

**Comment by hestonhoffman** [comment]
LGTM, I'll merge once the checks pass

**Pull request merged**